internal/worker/sshserver/session.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sshserver
+
+import (
+	"github.com/gliderlabs/ssh"
+)
+
+type stubSessionHandler struct{}
+
+func (s *stubSessionHandler) Handle(session ssh.Session, destinationAddress string) {
+	return
```

Comment:
do we need to close the session?

Reply:
That doesn't seem to be the way it should work. At least looking at the gliderlabs code, you don't close the session otherwise that would close the whole channel and the whole underlying tcp connection. And the client can be using that same channel for multiple things (in a normal ssh connection at least).

Reply:
Just an update, the default session handling code will run your handler then call Exit(0) which will return a exit code of 0 and then close the session.
In a follow-up PR, our handler will handle errors and perform Exit(1) on errors.